### ISSUE TITLE
[TESTING] Skull Quest

### ISSUE THREAD

**Issue by cambragol** [cambragol/advent-of-the-trinity#223]
The skull of Mondain quest has been tweaked, and will now suck the player into the Underdeep.

This quest needs testing for quality and balance.

**Comment by MrKnitster** [comment]
Just stumbled on this. I get sucked in, but have no idea where the skull now is. I suspect I am missing a clue somewhere along the way...

**Comment by cambragol** [comment]
It actually hasn't moved.

**Comment by cambragol** [comment]
except vertically.

**Comment by cambragol** [comment]
Did you manage to find the skull? I thought that it would be obvious that the skull was still where it was said to be, only 'lower'.

**Comment by MrKnitster** [comment]
Yes; once you get down there, it makes sense that the skull would have been sucked down too

**Comment by cambragol** [comment]
yup, that was the idea. I also thought that it would be a neat way to introduce the underdeep, as well as slide the Player into the Trinity Quests via the one truly optional Default side quest.

**Comment by cambragol** [comment]
What is the verdict on this change for you?

**Comment by MrKnitster** [comment]
Positive! There's nothing like that panic as the whirlpool takes you and then the relief that you survived and then the excitement of discovering the Underdeep! It reminds me of discovering Ambrosia in Ultima III

**Comment by MrKnitster** [comment]
Positive! There's nothing like that panic as the whirlpool takes you and then the relief that you survived and then the excitement of discovering the Underdeep! It reminds me of discovering Ambrosia in Ultima III
